SPECS/systemd/systemd.spec

```diff
@@ -43,6 +43,7 @@ BuildRequires:  kmod-devel
 BuildRequires:  libcap-devel
 BuildRequires:  libgcrypt-devel
 BuildRequires:  libselinux-devel
+BuildRequires:  libxcrypt-devel
```

Comment:
Does having a `BuildRequires` implicity (overwrite the old lib) link or we need to add an additional flag for linking?

Reply:
It's implicit - the build detects libcrypt in the environment from this package